src/components/DatePicker.vue

```diff
@@ -9,33 +9,28 @@
     <div slot="title" class="s-date-picker__title">{{ dateTitle }}</div>
     <vue-date-picker
       calendar-class="s-date-picker__calendar"
-      v-bind="{ ...datePickerProps }"
+      v-bind="{ ...$props }"
       :inline="true"
       @selected="updateDate"
       :initial-view="view"
       :minimum-view="view"
       :maximum-view="maxView"
-      :value="startDate"
+      :value="value"
     ></vue-date-picker>
   </pane-dropdown>
 </template>
 
 <script lang="ts">
 import { Component, Prop, Vue } from "vue-property-decorator";
 import PaneDropdown from "./PaneDropdown.vue";
-// import VueDatePicker from "vuejs-datepicker";
-
-interface selectedDate {
-  date: Date;
-  selected: boolean;
-}
+import VueDatePicker from "vuejs-datepicker";
 
 @Component({
   components: {
-    // VueDatePicker,
+    VueDatePicker,
     PaneDropdown
-  }
-  // props: { ...VueDatePicker.props }
+  },
+  props: { ...VueDatePicker.props }
```

Comment:
Curious, is this required because its a prop 'object' for the datepicker dependency?
why wont Prop() work for this?
again, just curious.